### ISSUE TITLE
Add Schema#serializable

### ISSUE THREAD

**Issue by jdegoes** [zio/zio-schema#19]
Add a new method to Schema:

```scala
sealed trait Schema[A] {
  def serializable: Schema[Schema[_]]
}
```

This will return a `Schema` for the original `Schema`, but without the transform nodes -- i.e. without any functions inside of them. 

This will allow you to serialize a schema across the wire (or deserialize it), as long as you're willing to "forget" which Scala types are being used to model the built-in types.

**Comment by aphoristicartist** [comment]
Hi John. 
Will it be correct adding additional case for Schema sum type like:
```scala
sealed case class SuperSchema(schema: Schema[_]) extends Schema[Schema[_]]
```
And then pattern match for each type like this:
 ```scala
 def serializable: Schema[Schema[_]] =
    self match {
    case p: Schema.Primitive[_] => Schema.SuperSchema(p)
    case Schema.Transform(codec, _, _) => Schema.SuperSchema(codec)
    case _ => ???
  }
  ```
Thank you

**Comment by jdegoes** [comment]
Hi @aphoristicartist ,

In this case, you do not need an additional case. 

You can pattern match on "self", and recurse, stripping out all Transform nodes. This will give you a Schema, but without Transform. Then given that Schema, you can, in turn, create another Schema that describes it. 

The first step of this problem should be straightforward: create a method called 'withoutTransform' on 'Schema' that returns `Schema[_]`, and it will be the "same" schema, but without any `Transform` nodes. It has to return a wildcard type because you don't know what the type will be without the `Transform` nodes. 

The next part of the problem, describing the schema of THAT schema, is a little mind-bending. But it will be easier when the first step is completed.

**Comment by thinkharderdev** [comment]
Closed by #83
